sdk/appconfiguration/app-configuration/samples/react-sample/sample.env

```
@@ -1 +1,14 @@
-REACT_APP_APPCONFIG_CONNECTION_STRING=<connection-string>
+# Rename this to .env in order to have React automatically
+# pull in all these variables (anything that starts with REACT_APP_ will be
+# made available. See the documentation for adding custom environment variables 
+# at the following link: https://create-react-app.dev/docs/adding-custom-environment-variables/)
+
+# App Config Endpoint to be obtained from the portal
+REACT_APP_APP_CONFIG_ENDPOINT=<https://account-name.azconfig.io>
```

I think we typically just use 'APPCONFIG' in our sample.env files: (ie: `APPCONFIG_CONNECTION_STRING`). 

```suggestion
REACT_APP_APPCONFIG_ENDPOINT=<https://account-name.azconfig.io>
```